.env-example

```
@@ -1 +1,2 @@
 DISCORD_BOT_TOKEN=xxxxxxxx
+CALENDAR_WEBHOOK=xxxxxxxx
```

```suggestion
CALENDAR_WEBHOOK_URL=xxxxxxxx

```
末尾改行ほしいな〜と思ったけど、.editorconfig で insert_final_newline を false にされちゃってるんだ。
後で .editorconfig を修正するので、こちらも改行入れておいてほしい

あと、末尾に URL って入れたほうが、.env-example を見た時にわかりやすいと感じた。どうかな？

---

既にあるEditorconfigの設定を変えるようにここで言うのは off-topic な感じある。他のファイルにも影響あるだろうしPR分けて欲しい。

---

editorconfig は後で別の PR で変えるから、正しいとされる末尾改行にしておいて欲しい って感じだった。
PR 変える話については同意